CHANGELOG.md

```diff
@@ -4,6 +4,23 @@
 
 - Add conditional logic checking for json-logic ([#41](https://github.com/remoteoss/json-schema-form/pull/41)) ([6292b01e](https://github.com/remoteoss/json-schema-form/commit/6292b01e3f77a9038328d7375080ffc4cb30dbc8))
 
+###### Full API additions now supported from 0.6.5 onwards.
+
+
+- New custom JSON Schema keyword `x-jsf-logic` added to support cross-field validations.
+- `x-jsf-logic` can contain:
+
+  - `validations` - JsonLogic rules that validate fields and return booleans
+  - `computedValues` - JsonLogic rules that compute dynamic values
+  - `allOf.if/then/else` - Conditional logic using validations and computedValues
+
```

Comment:
What do you think of adding a line that points to the existing tests and examples to learn more? It's far from ideal (proper docs) but it's already a good starting)